clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -432,10 +433,14 @@ protected void handleTagCommand() {
     String url = getUrl();
     FullName name = new FullName(line);
     String metalake = name.getMetalakeName();
-    String tag = line.getOptionValue(GravitinoOptions.TAG);
-
+    String[] tags =
+        Arrays.stream(line.getOptionValues(GravitinoOptions.TAG)).distinct().toArray(String[]::new);
+    if (!validateMultipleTagOption(tags)) {
+      System.err.println(ErrorMessages.MULTIPLE_TAG_COMMAND_ERROR);
+      return;
+    }
     if (CommandActions.DETAILS.equals(command)) {
-      new TagDetails(url, ignore, metalake, tag).handle();
+      new TagDetails(url, ignore, metalake, tags[0]).handle();
```

Comment:
There is no guarantee that tags is not empty.

Reply:
> There is no guarantee that tags is not empty.

fixed